protocol/streaming/grpc/grpc_streaming_manager.go

```diff
@@ -141,6 +141,63 @@ func (sm *GrpcStreamingManagerImpl) SendOrderbookUpdates(
 	}
 }
 
+// SendOrderbookFillUpdates groups fills by their clob pair ids and
+// sends messages to the subscribers.
+func (sm *GrpcStreamingManagerImpl) SendOrderbookFillUpdates(
```

Comment:
similar looking functions - worth to create helper?

nbd - will let you make the call